pendulum:
  # Publish all joint states
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Controllers
  FLwheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_front_left_wheel
    pid: {p: 10.0, i: 0.0, d: 0.0}
  
  FRwheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_front_right_wheel
    pid: {p: 10.0, i: 0.0, d: 0.0}

  BLwheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_back_left_wheel
    pid: {p: 10.0, i: 0.0, d: 0.0}

  BRwheel_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_back_right_wheel
    pid: {p: 10.0, i: 0.0, d: 0.0}
